lib/MusicBrainz/Server/Controller.pm

```diff
@@ -299,4 +299,12 @@ sub error {
     $c->detach;
 }
 
+sub ws1_gone : Chained('/') PathPart('ws/1') {
+    my ($self, $c) = @_;
+
+    $c->res->content_type('text/plain; charset=utf-8');
+    $c->res->body('https://blog.metabrainz.org/2018/02/01/web-service-ver-1-0-ws-1-will-be-removed-in-6-months/');
```

Comment:
This could probably be done on the gateways, and maybe a different response is more appropriate. (Note that /ws/1 did return errors as text/plain in general, for some reason.)

Reply:
A simple "/ws/1 is dead since ..." message with proper error code (503) should do. And yes, it should be done on gateways.

Reply:
Yup, 410 Gone is even better of course